### PR TITLE
Update `txnbuild` go doc

### PR DESCRIPTION
**Pull request by Kafakk**

### CODE REVIEW

File: libs/txnbuild/mint_credit.go [hunk 1]
```diff
@@ -38,6 +40,7 @@ func (mintCredit *MintCredit) BuildXDR() (vxdr.VeloOp, error) {
 	return vxdr.VeloOp{Body: body}, nil
 }
 
+// FromXDR for MintCredit initialises the vtxnbuild struct from the corresponding xdr Operation.
```
Comment (toppy21) [inline libs/txnbuild/mint_credit.go]: FromXDR for MintCredit initialises the vtxnbuild struct from the corresponding **XDR** Operation.

File: libs/txnbuild/mint_credit.go [hunk 2]
```diff
@@ -50,6 +53,8 @@ func (mintCredit *MintCredit) FromXDR(vXdrOp vxdr.VeloOp) error {
 	return nil
 }
 
+// Validate for MintCredit validates the required struct fields. It returns an error if any of the fields are
```
Comment (toppy21) [inline libs/txnbuild/mint_credit.go]: Validation function for MintCredit. Validates the required struct fields. It returns an error if any of the fields are

File: libs/txnbuild/price_update.go [hunk 3]
```diff
@@ -50,6 +53,8 @@ func (priceUpdate *PriceUpdate) FromXDR(vXdrOp vxdr.VeloOp) error {
 	return nil
 }
 
+// Validate for PriceUpdate validates the required struct fields. It returns an error if any of the fields are
```
Comment (toppy21) [inline libs/txnbuild/price_update.go]: Validation function for PriceUpdate. Validates the required struct fields. It returns an error if any of the fields are

File: libs/txnbuild/rebalance_reserve.go [hunk 4]
```diff
@@ -20,6 +22,7 @@ func (rebalanceReserve *RebalanceReserve) BuildXDR() (vxdr.VeloOp, error) {
 	return vxdr.VeloOp{Body: body}, nil
 }
 
+// FromXDR for RebalanceReserve initialises the vtxnbuild struct from the corresponding xdr Operation.
```
Comment (toppy21) [inline libs/txnbuild/rebalance_reserve.go]: FromXDR for RebalanceReserve initialises the vtxnbuild struct from the corresponding **XDR** Operation.

File: libs/txnbuild/price_update.go [hunk 5]
```diff
@@ -37,6 +39,7 @@ func (priceUpdate *PriceUpdate) BuildXDR() (vxdr.VeloOp, error) {
 	return vxdr.VeloOp{Body: body}, nil
 }
 
+// FromXDR for PriceUpdate initialises the vtxnbuild struct from the corresponding xdr Operation.
```
Comment (toppy21) [inline libs/txnbuild/price_update.go]: FromXDR for PriceUpdate initialises the vtxnbuild struct from the corresponding **XDR** Operation.

File: libs/txnbuild/rebalance_reserve.go [hunk 6]
```diff
@@ -29,6 +32,8 @@ func (rebalanceReserve *RebalanceReserve) FromXDR(vXdrOp vxdr.VeloOp) error {
 	return nil
 }
 
+// Validate for RebalanceReserve validates the required struct fields. It returns an error if any of the fields are
```
Comment (toppy21) [inline libs/txnbuild/rebalance_reserve.go]: Validation function for RebalanceReserve. Validates the required struct fields. It returns an error if any of the fields are

File: libs/txnbuild/redeem_credit.go [hunk 7]
```diff
@@ -42,6 +44,7 @@ func (redeemCredit *RedeemCredit) BuildXDR() (vxdr.VeloOp, error) {
 	return vxdr.VeloOp{Body: body}, nil
 }
 
+// FromXDR for RedeemCredit initialises the vtxnbuild struct from the corresponding xdr Operation.
```
Comment (toppy21) [inline libs/txnbuild/redeem_credit.go]: FromXDR for RedeemCredit initialises the vtxnbuild struct from the corresponding **XDR** Operation.

File: libs/txnbuild/redeem_credit.go [hunk 8]
```diff
@@ -55,6 +58,8 @@ func (redeemCredit *RedeemCredit) FromXDR(vXdrOp vxdr.VeloOp) error {
 	return nil
 }
 
+// Validate for RedeemCredit validates the required struct fields. It returns an error if any of the fields are
```
Comment (toppy21) [inline libs/txnbuild/redeem_credit.go]: Validation function for RedeemCredit. Validates the required struct fields. It returns an error if any of the fields are

File: libs/txnbuild/setup_credit.go [hunk 9]
```diff
@@ -38,6 +40,7 @@ func (setupCredit *SetupCredit) BuildXDR() (vxdr.VeloOp, error) {
 	return vxdr.VeloOp{Body: body}, nil
 }
 
+// FromXDR for SetupCredit initialises the vtxnbuild struct from the corresponding xdr Operation.
```
Comment (toppy21) [inline libs/txnbuild/setup_credit.go]: FromXDR for SetupCredit initialises the vtxnbuild struct from the corresponding **XDR** Operation.

File: libs/txnbuild/setup_credit.go [hunk 10]
```diff
@@ -51,6 +54,8 @@ func (setupCredit *SetupCredit) FromXDR(vXdrOp vxdr.VeloOp) error {
 	return nil
 }
 
+// Validate for SetupCredit validates the required struct fields. It returns an error if any of the fields are
```
Comment (toppy21) [inline libs/txnbuild/setup_credit.go]: Validation function for SetupCredit. Validates the required struct fields. It returns an error if any of the fields are

File: libs/txnbuild/whitelist.go [hunk 11]
```diff
@@ -36,6 +38,7 @@ func (whitelist *Whitelist) BuildXDR() (vxdr.VeloOp, error) {
 	return vxdr.VeloOp{Body: body}, nil
 }
 
+// FromXDR for Whitelist initialises the vtxnbuild struct from the corresponding xdr Operation.
```
Comment (toppy21) [inline libs/txnbuild/whitelist.go]: FromXDR for Whitelist initialises the vtxnbuild struct from the corresponding **XDR** Operation.

File: libs/txnbuild/whitelist.go [hunk 12]
```diff
@@ -49,6 +52,8 @@ func (whitelist *Whitelist) FromXDR(vXdrOp vxdr.VeloOp) error {
 	return nil
 }
 
+// Validate for Whitelist validates the required struct fields. It returns an error if any of the fields are
```
Comment (toppy21) [inline libs/txnbuild/whitelist.go]: Validation function for Whitelist. Validates the required struct fields. It returns an error if any of the fields are

**Review by athiwatp — APPROVED** [review]

**Pull request merged**